lib/node_modules/@stdlib/math/base/special/sincospi/benchmark/julia/benchmark.jl

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env julia
+
+import BenchmarkTools
+
+# Benchmark variables:
+name = "sincospi";
+repeats = 3;
+
+"""
+	print_version()
+
+Prints the TAP version.
+
+# Examples
+
+``` julia
+julia> print_version()
+```
+"""
+function print_version()
+	@printf( "TAP version 13\n" );
+end
+
+"""
+	print_summary( total, passing )
+
+Print the benchmark summary.
+
+# Arguments
+
+* `total`: total number of tests
+* `passing`: number of passing tests
+
+# Examples
+
+``` julia
+julia> print_summary( 3, 3 )
+```
+"""
+function print_summary( total, passing )
+	@printf( "#\n" );
+	@printf( "1..%d\n", total ); # TAP plan
+	@printf( "# total %d\n", total );
+	@printf( "# pass  %d\n", passing );
+	@printf( "#\n" );
+	@printf( "# ok\n" );
+end
+
+"""
+	print_results( iterations, elapsed )
+
+Print benchmark results.
+
+# Arguments
+
+* `iterations`: number of iterations
+* `elapsed`: elapsed time (in seconds)
+
+# Examples
+
+``` julia
+julia> print_results( 1000000, 0.131009101868 )
+```
+"""
+function print_results( iterations, elapsed )
+	rate = iterations / elapsed
+
+	@printf( "  ---\n" );
+	@printf( "  iterations: %d\n", iterations );
+	@printf( "  elapsed: %0.9f\n", elapsed );
+	@printf( "  rate: %0.9f\n", rate );
+	@printf( "  ...\n" );
+end
+
+"""
+	sincospi( x )
+
+Compute the sine and cosine of a number (since sincospi is not directly exposed in julia).
+
+# Arguments
+
+* `x`: argument, in radians
+
+# Examples
+
+``` julia
+julia> sincospi( 0.0 )
+```
+"""
+function sincospi( x )
+	[ sin( pi * x ), cos( pi * x ) ];
```

Comment:
You should be able to use the Julia functions `sinpi` and `cospi` here.

Reply:
Ah, forgot that was exposed. Got it mixed up with `sincos` which is the one that's implemented but not exposed.